### DISCUSSION TITLE
Use a Favicon on my Neuron Website

### DISCUSSION THREAD

**Discussion by misudashi** [srid/neuron#672] [General]
Hello everyone. I was wondering how I could use a Favicon on my Neuron Website. Thank you for your answers !

**Comment by srid** [comment]
@misudashi See the 'favicon' section in https://neuron.zettel.page/customize-site

Note that neuron is no longer in development; all new development happens in Emanote, neuron's successor, which also supports neuron like layout: https://emanote.srid.ca/demo/neuron-layout
